src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -1966,6 +1966,63 @@ bool isCondValueInContextElementResponse(ConditionValueList* condValues, Context
 }
 
 
+
+/* ****************************************************************************
+*
+* someEmptyCondValue -
+*
+* This logic would be MUCH simpler in the case conditions was a single field instead of a vector.
```

Comment:
Is this something we are considering?
Otherwise, the comment is not very interesting ...